linux_os/guide/system/accounts/accounts-session/accounts_tmout/ansible/shared.yml

```diff
@@ -5,4 +5,22 @@
 # disruption = low
 {{{ ansible_instantiate_variables("var_accounts_tmout") }}}
 
-{{{ ansible_only_lineinfile(path='/etc/profile.d/tmout.sh', line_regex='TMOUT=', new_line='declare -xr TMOUT={{ var_accounts_tmout }}', create='yes') }}}
+{{% if product in ["ol7", "rhel7"] %}}
```

Comment:
I find product-specific conditionals difficult to work with, because you have to analyze them again if you think that another product should be included there.
What about using an explanatory jinja variable like we did for the [grub template](https://github.com/ComplianceAsCode/content/blob/master/shared/templates/grub2_bootloader_argument/oval.template)? I guess that here it could be named somehow like `system_configuration_using_etc_bashrc_expected`.

Reply:
Updated.